setup.cfg

```
@@ -52,7 +52,7 @@ install_requires =
 
 [options.entry_points]
 console_scripts =
-    django = development.cli:django
+    concrete-datastore = development.cli:django
```

I would like to keep the current entry_points valid

```suggestion
    django = development.cli:django
    concrete-datastore = development.cli:django
```